portfolio/src/main/java/com/google/sps/data/Credentials.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
+
+/** A user's credentials. */
+public final class Credentials {
+
+  private final String nickname;
+  private final String authUrl;
```

Comment:
Nit: in google java style we tend to avoid abbreviations in variable names. Use the whole thing. 

Notably, auth is especially bad because it could authentication or it could be authorization (which are distinct but related concepts).

Reply:
Change variable name.